int8    Mode
string  FilePath
---
int8    Status

